agent/api/task/task.go

```diff
@@ -2538,3 +2539,12 @@ func (task *Task) GetContainerIndex(containerName string) int {
 	}
 	return -1
 }
+
+func getDomainForPartition(region string) string {
+	partition, ok := endpoints.PartitionForRegion(endpoints.DefaultPartitions(), region)
+	if !ok {
+		seelog.Warnf("No partition resolved for region (%s). Using AWS default (%s)", region, endpoints.AwsPartition().DNSSuffix())
```

Comment:
we should probably add context here that this is for EFS.

Reply:
Was thinking this could be used for other things in future

Reply:
Maybe not?

Reply:
yeah, we can change it later if needed. Just that I would be surprised to see this message without much context. also, do we know when this error would occur?